system-tests/CFortranInterop/expected.fab.txt

```diff
@@ -0,0 +1,30 @@
+system-tests/CFortranInterop/c_lower.c
+    hash: 2457312520
+system-tests/CFortranInterop/c_roundtrip.c
+    hash: 3255455825
+system-tests/CFortranInterop/f_inters.F90
+    hash: 1710816464
+system-tests/CFortranInterop/f_inters.h
+    hash: 3170915542
+system-tests/CFortranInterop/f_var.F90
+    hash: 22378257
+system-tests/CFortranInterop/f_var.h
+    hash: 58283018
+f_inters
+    found in: system-tests/CFortranInterop/working/f_inters.f90
+    depends on: ['get_f_var_ptr']
+f_var
+    found in: system-tests/CFortranInterop/working/f_var.f90
+    depends on: []
+f_inter
+    found_in: system-tests/CFortranInterop/working/f_inters.f90
+    depends on: []
+get_f_var_ptr
+    found_in: system-tests/CFortranInterop/working/c_lower.c
+    depends on: ['helloworld']
+helloworld
+    found_in: system-tests/CFortranInterop/working/f_var.f90
+    depends on: []
+main
+    found_in: system-tests/CFortranInterop/working/c_roundtrip.c
+    depends on: ['f_inter']
```

Comment:
Not for this ticket but the fact that fab stdout is almost identical to the dump output seems unnecessary. Now that we have some logging output could we be testing that instead?